.github/workflows/test.yml

```diff
@@ -0,0 +1,63 @@
+# documentation: https://help.github.com/en/articles/workflow-syntax-for-github-actions
+name: Tests for EESSI test suite, using EESSI pilot repo
+on: [push, pull_request, workflow_dispatch]
+permissions: read-all
+jobs:
+  test_with_eessi_pilot:
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        EESSI_VERSION:
+        - "2021.12"
+    steps:
+        - name: Check out software-layer repository
+          uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+          with:
+            persist-credentials: false
+
+        - name: Mount EESSI CernVM-FS pilot repository
+          uses: eessi/github-action-eessi@58b50fd2eead2162c2b9ac258d4fb60cc9f30503  # v2.0.13
+          with:
+            eessi_stack_version: ${{matrix.EESSI_VERSION}}
+
+        - name: Run test suite
+          run: |
+            # install latest version of EasyBuild, to install ReFrame with it,
+            # since that includes the ReFrame test library (hpctestlib) that we rely on
+            pip3 install --user easybuild
+            export PATH=$HOME/.local/bin:$PATH
+            eb --version
+            export EASYBUILD_PREFIX=$HOME/easybuild
+            # need to force module generation with --module-only --force because 'pip check' fails
+            # in EESSI pilot 2021.12, see https://github.com/EESSI/compatibility-layer/issues/152
+            eb ReFrame-4.2.0.eb || eb ReFrame-4.2.0.eb --module-only --force
+
+            # load ReFrame
+            module use $HOME/easybuild/modules/all
+            module load ReFrame/4.2.0
+            reframe --version
+
+            # configure ReFrame (cfr. https://reframe-hpc.readthedocs.io/en/stable/manpage.html#environment)
+            export RFM_CONFIG_FILES=$PWD/eessi/reframe/config/github_actions.py
+            export RFM_SYSTEM=github_actions_eessi
+            export RFM_CHECK_SEARCH_PATH=$PWD/eessi/reframe/eessi_checks/applications
+
+            # deliberately passing path to configuration file via --config-file because that yields better errors
+            # if configuration file can not be parsed;
+            # only setting $RFM_CONFIG_FILES results in silently ignoring the configuration file...
```

Comment:
weird, i cannot reproduce it.

Reply:
May have been a bug in an old ReFrame version, I was accidentally using the old ReFrame version from the EESSI 2021.12 pilot version halfway through getting this to work, and there it was totally silent on the broken configuration file when using `$RFM_CONFIG_FILES` rather than `--config-file` or `-C`

Reply:
fixed in 87b8efb